OpenRA.Mods.Common/ServerTraits/LobbyCommands.cs

```diff
@@ -1442,7 +1442,7 @@ static string MissionBriefingOrDefault(S server)
 		{
 			var missionData = server.Map.WorldActorInfo.TraitInfoOrDefault<MissionDataInfo>();
 			if (missionData != null && !string.IsNullOrEmpty(missionData.Briefing))
-				return missionData.Briefing.Replace("\\n", "\n");
+				return server.Map.GetLocalisedString(missionData.Briefing).Replace("\\n", "\n");
```

Comment:
This doesn't look right. The servers should be sending the translation id so the clients can do the translation - like for server messages.